sql-statements/sql-statement-alter-sequence.md

```diff
@@ -0,0 +1,214 @@
+---
+title: ALTER SEQUENCE
+summary: An overview of the usage of ALTER SEQUENCE for the TiDB database.
+---
+
+# ALTER SEQUENCE
+
+The `ALTER SEQUENCE` statement alters sequence objects in TiDB. The sequence is a database object that is on a par with the `Table` and the `View` object. The sequence is used to generate serialized IDs in a customized way.
+
+## Synopsis
+
+```ebnf+diagram
+CreateSequenceStmt ::=
+    'ALTER' 'SEQUENCE' TableName CreateSequenceOptionListOpt
+
+TableName ::=
+    Identifier ('.' Identifier)?
+
+CreateSequenceOptionListOpt ::=
+    SequenceOption*
+
+SequenceOptionList ::=
+    SequenceOption
+
+SequenceOption ::=
+    ( 'INCREMENT' ( '='? | 'BY' ) | 'START' ( '='? | 'WITH' ) | ( 'MINVALUE' | 'MAXVALUE' | 'CACHE' ) '='? ) SignedNum
+|   'COMMENT' '='? stringLit
+|   'NOMINVALUE'
+|   'NO' ( 'MINVALUE' | 'MAXVALUE' | 'CACHE' | 'CYCLE' )
+|   'NOMAXVALUE'
+|   'NOCACHE'
+|   'CYCLE'
+|   'NOCYCLE'
+|   'RESTART' ( ( '='? | 'WITH' ) SignedNum )?
+```
+
+## Syntax
+
+{{< copyable "sql" >}}
+
```

Comment:
```suggestion
```

Reply:
The docs website now supports providing the copy button without this copyable line